app/javascript/components/WorkflowsPage/WorkflowForm.tsx

```diff
@@ -296,7 +310,7 @@ const WorkflowForm = () => {
       navigation={workflow ? <EditPageNavigation workflowExternalId={workflow.external_id} /> : null}
       actions={
         <>
-          <Link to="/workflows" className="button" inert={isSaving}>
+          <Link href={Routes.workflows_url()} className="button" inert={isSaving || undefined}>
```

Comment:
It's good to replace hard-coded paths with our url helpers. But use `_path` and not `url`. We shouldn't generate the full URL unless necessary.
```suggestion
          <Link href={Routes.workflows_path()} className="button" inert={isSaving || undefined}>
```